test/clt-tests/core/test-fuzzy-search2.rec

```diff
@@ -0,0 +1,33 @@
+––– block: ../base/start-searchd –––
+––– input –––
+php -d memory_limit=-1 ./test/clt-tests/comparison-overhead-json-sql/load_us_names.php 10000 10 1000000 1 > /dev/null
+––– output –––
+––– input –––
+mysql -h0 -P9306 -e "SELECT COUNT(*) FROM name;"
+––– output –––
++----------+
+| count(*) |
++----------+
+|  1000000 |
++----------+
+––– input –––
+mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('RICH') OPTION fuzzy=-1;"
+––– output –––
+ERROR 1064 (42000) at line 1: P01: syntax error, unexpected '-' near '-1'
+––– input –––
+mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('SMITH') OPTION fuzzy=`0`;"
```

Comment:
` should be escaped cuz 0 executed by bash as command, thats why we have error bash command not found